circleci/circleci.sh

```diff
@@ -0,0 +1,21 @@
+DEFAULT_LANG="ruby"
+lang=$DEFAULT_LANG
+
+detectProjectLang () {
```

Comment:
I was thinking about deleteing this bash function and leaving if's in normal block. Bash functions are stupid, without arguments, and they are not really necessary here cause we call this function right after definiotion. WDYT ?